files/src/main/java/io/luna/files/AbstractJsonFileParser.java

```diff
@@ -1,4 +1,4 @@
-package io.luna.util.parser;
+package io.luna.files;
```

Comment:
Isn't it convention to have package names be singular (`io.luna.file`) instead of plural (`io.luna.files`)?

Reply:
It depends on what it's being used for. 

Directories/packages with a plural name are usually reserved for collections like `docs`, `services`, and `contexts`. You would expect there to be multiple documents, services, and contexts in each package respectively.

Directories/packages whos name is singular usually refers to an abstraction, like `event`. You're likely to find objects like `EventListener`, `EventDispatcher`, and `EventManager` within that package.

In this case I've named the module `files` because the project supports reading/writing to multiple file types (`.json`, `.txt`, and `.toml`) and it made more sense to me to have the _collection_ of read and write abstractions as a single module.